### DISCUSSION TITLE
Татуировки

### DISCUSSION THREAD

**Discussion by dosinabox** [dosinabox/g2nr_unofficial_update#434] [General]
В Г2 у Лестера пропали татуировки и изменились доспехи. Нужно ли заморачиваться с возвращением оригинальной внешности из Г1? Понадобятся две новые текстуры и новая модель доспехов.
![Screenshot_1](https://user-images.githubusercontent.com/39790055/77991913-acea9a00-732d-11ea-8035-af3c441b387e.jpg)

**Comment by Kor-Angar** [comment]
Татуировки на теле пропали и у Кор Ангара. У Лестера в Г1 были простые доспехи, хотя по соц статусу на мой взгляд должны были быть тяжёлые. Поэтому то что в Г2 у него тяжёлые доспехи мне кажется это наоборот хорошо 😀 ведь этот доспех переносили спициально для Лестера 😀А вот тело с татуировкой надо вернуть и Лестера и Кор Ангара и Фртуно. У Кор Ангара на шее была видна тату при одетом доспези 🙃8:56, 31 марта 2020 г., dosinabox <notifications@github.com>:
В Г2 у Лестера пропали татуировки и изменились доспехи. Нужно ли заморачиваться с возвращением оригинальной внешности из Г1? Понадобятся две новые текстуры и новая модель доспехов.



—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.
-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by Kor-Angar** [comment]
а ещё у него Overlay поменялся :-(

```
	Mdl_ApplyOverlayMds(self,"Humans_Relaxed.mds"); // Gothic 1
//	Mdl_ApplyOverlayMds(self,"Humans_Mage.mds"); // Gothic 2
```

**Comment by Kor-Angar** [comment]
Я Кор Ангару дёрнул шкурку в отдельную так как спящий послал мне знак
![image](https://user-images.githubusercontent.com/49514764/78086184-db1bb880-73c5-11ea-901d-7b3b879a130d.png)

кстати такое же тело было ещё у двух  стражников в СЛ, но ElderGamer почему то решил что это неправильно.. облико морали:-( хотя на зоне бить татушки вроде это норм.. и в море тоже:-) и в армии:-) эта такая специфика закрытого  мужского коллектива :-)
![image](https://user-images.githubusercontent.com/49514764/78086695-69dd0500-73c7-11ea-823f-c18a2e0f92cf.png)

и тут подумал о бандитах в НВ, которые сбежали с зоны. Там есть безымянный бандит который тренирует силу с голым торсом и топором орков.. он готовит еду. ты ему ещё какое то имя дала.. так вот, может ему дать тело с татушкой? :-) смотреться наверное будет брутально :-)


> а ещё у него Overlay поменялся :-(

а в G1_MDK Humans_Mage :-))
![image](https://user-images.githubusercontent.com/49514764/78086909-fdaed100-73c7-11ea-8003-6d39aaddb129.png)


тело с татушками как у Лестера оказалось ещё у двух Темплаеров и у 4(!) воров из НЛ, которые в Г2НВ являются уже бандитами.
![image](https://user-images.githubusercontent.com/49514764/78087755-c261d180-73ca-11ea-8584-a78788cafc05.png)
 правда им ElderGamer  тоже татушки потёр. Декриминализация :-) но  в целом история с ТаТу вполне актуально.

на форумах конечно скажут что тату там про спящего и с чего бы ворам их набивать? с тоже наверное почему люди не знающего японского набивают себе японские иероглифы. тема тату сложна и имеет большие бэки и в истории и в культуре :-)

p.s.
Шакал тоже был с Тату :-)
![image](https://user-images.githubusercontent.com/49514764/78088904-4c5f6980-73ce-11ea-9fa9-98802a5d074a.png)

p.s.s.
млин.. может я не так ищу.. наверное пол колонии в тату.
![image](https://user-images.githubusercontent.com/49514764/78089318-c2b09b80-73cf-11ea-9a7a-745450dd1a5a.png)
хотя у рудокопов нет. это и логично, им не положено. вот как станет Вором, так сразу татушку старшие товарищи и набьют :-)
надо будет не забыть спросить у ElderGamer служил ли он в армии :-) а то у него все наёмники, вместе с Ли решились тату :-)


****
вообщем всё очень просто оказалось
![image](https://user-images.githubusercontent.com/49514764/78089653-bda01c00-73d0-11ea-9fb1-47687e72ab1d.png)
2,3,4 это трудяги - рудокопы
1 - это ребята с тату, тату видимо подчеркивает их соц статус
0 - ребята без тату
ну и в 1 входят ещё сектанты, они все в тату :-)

сделал у себя татухи :-)
![image](https://user-images.githubusercontent.com/49514764/78090175-10c69e80-73d2-11ea-8136-3fbb2bde2a55.png)

**Comment by Kor-Angar** [comment]
у Призраков по моему тоже нет Тату.  тату есть у Стражников и Наёмников. а так же у нескольких Воров. ну и все в БЛ :-)

**Comment by Kor-Angar** [comment]
https://www.youtube.com/watch?v=U0HeQRd27kk

**Comment by Kor-Angar** [comment]
а есть оказывается и Темплаеры без тату :-)
![image](https://user-images.githubusercontent.com/49514764/78095340-576ec580-73df-11ea-801c-279c728c8500.png)

**Comment by Kor-Angar** [comment]
![image](https://user-images.githubusercontent.com/49514764/78095361-65bce180-73df-11ea-8978-007e144510a6.png)
![image](https://user-images.githubusercontent.com/49514764/78095370-6bb2c280-73df-11ea-96bf-812245188241.png)
![image](https://user-images.githubusercontent.com/49514764/78095381-73726700-73df-11ea-927d-1de4f5d0b150.png)

**Comment by dosinabox** [comment]
> А вот тело с татуировкой надо вернуть и Лестера и Кор Ангара и Фртуно.

Для Фортуно разработчики добавили отдельную текстуру тела в аддоне.
Лестер и Ангар теперь тоже будут со своими татуировками из Г1. Пришлось добавить две новые текстуры, потому что у всех этих трех бывших сектантов разный цвет кожи.

**Comment by Kor-Angar** [comment]
> Для Фортуно разработчики добавили отдельную текстуру тела в аддоне.

в Готике он был "европейцем с тату"
`
Mdl_SetVisualBody(self,"hum_body_Naked0",1,1,"Hum_Head_Bald",27,1,nov_armor_h);
`
Фортуно стал Латиносом?	
***
может тогда и 4 тело с тату добавить? оно было у генерала Ли и многих наёмников :-)

***
по идеи их бы назвать осмысленно и  добавить в AI_Constants.d **например**

```
const int BodyTex_P_T	= xx;	//Pale Tätowierung 	// G1	"hum_body_Naked0",1,0
const int BodyTex_N_T	= xx;	//Normal Tätowierung	// G1	"hum_body_Naked0",1,1
const int BodyTex_L_T	= 10;	//Latino Tätowierung	// G1	"hum_body_Naked0",1,2
const int BodyTex_B_T	= xx;	//Black Tätowierung	// G1	"hum_body_Naked0",1,3
```

***

было бы здорово добавить в твой примиритель голов возможность посмотреть как те или иные будут смотреться с татухами.  в частности я бы попробовал дать пару тату кому нибудь  например

  | G1 | G2 NotR
-- | -- | --
BodyTex_B_T | Кор Ангар | Пират Алигатор Джек,
BodyTex_L_T | - | Бандит Эстебан
BodyTex_N_T | Фортуно , Ли | пару безымянных наёмников, наёмник Engardo
BodyTex_P_T | Лестер | Пират Грег или Пират Генри

в целом Тату по смыслу подходит Наёмникам, Пиратам и Бандитам. конечно это зависит от того как они смотрятся с доспехами. например с тяжелыми доспехами Наёма и доспехами "гвардии Равена" они смотрятся "незаметно" :-)

Среди Бандитов это могут быть например Хуно, Эстебан, "гвардейцы" Равена,

**Comment by Kor-Angar** [comment]
upd пост

**Comment by dosinabox** [comment]
> Фортуно стал Латиносом?

Никак нет. Он "pale" (бледный). Просто в Г2 другая нумерация текстур тел.

> может тогда и 4 тело с тату добавить? оно было у генерала Ли и многих наёмников :-)

У Ли такой же цвет кожи, как и у Фортуно.

**Comment by dosinabox** [comment]
А есть ли татуировки у Ли в Г3?

**Comment by Kor-Angar** [comment]
Не знаю. в Готику 3 не играл. -- Отправлено из мобильного приложения Яндекс.Почты

**Comment by Kor-Angar** [comment]
https://worldofplayers.ru/threads/35139/

**Comment by dosinabox** [comment]
Получается, что именные персонажи с татуровками из Г1, которые есть в Г2 - это только Лестер, Фортуно, Ангар и Ли?

**Comment by Kor-Angar** [comment]
вроде да.

![image](https://user-images.githubusercontent.com/49514764/78192146-39a56d00-7480-11ea-8a20-3dc2263051a0.png)

**Comment by dosinabox** [comment]
Должны ли быть татуировки у бывшего генерала короля и наемника - вопрос с точки зрения канона, но пока верну как было в первой части.

В итоге: татуировки получили Лестер, Ангар и Ли, а в файлах игры теперь есть текстуры всех тел с татуировками, кроме латиносов. При желании можно будет применить ещё кому-нибудь необязательному.

**Comment by Kor-Angar** [comment]
А как же латинасы? Это дискриминация по национальному признаку 😀

**Comment by dosinabox** [comment]
Сюжетных именных персонажей из Г1 с татуировками вроде нет, а значит нечего раздувать вес мода лишней текстурой)

**Comment by Kor-Angar** [comment]
поделись тяжелыми текстурами с тату, а то они у меня какие все маленькие по 170 Кб.

**Comment by dosinabox** [comment]
У меня тоже, это обычный вес для такого разрешения.

**Comment by Kor-Angar** [comment]
вон сколько латиносов бандитов, а ты про 170 кб говоришь
![image](https://user-images.githubusercontent.com/49514764/78220837-f1f80300-74ca-11ea-87da-cbe63760dfb9.png)

тут явно замешано что то личное :-)

**Comment by Kor-Angar** [comment]
Добавь пожалуйста BodyTex_Tattoo_L . а то как то не полюдски получается: латиносов обделят татушками. опять какая то сегрегация.

const int BodyTex_Tattoo_L = XX;	//Latino Tätowierung	// G1	"hum_body_Naked0",1,2

***

а я спокойно сделаю сводную таблицу с предложением кому дать татушки дополнительно(некоторые бандиты, наёмники, пираты), исходя из той логике что была заложена в Готике 1. протестирую и будет наглядно видно насколько это корректно.

**Comment by dosinabox** [comment]
У черного мага на Ирдорате видны руки и шея, поэтому он тоже получает татуировки

![image](https://user-images.githubusercontent.com/39790055/95664307-4b627e00-0b4f-11eb-9306-060a4c837ceb.png)

**Comment by Kor-Angar** [comment]
@dosinabox  а можешь в дискуссию конвертировать? я перечитаю что писал, посмотрю что делал в своей сборке и может ввиде мини-мода опублику. Татухи очень важны для мальчиков. Особенно в юности. Это же не татуаж :-)

**Reply by Kor-Angar** [reply]
спс
